variables.tf

```diff
@@ -36,6 +36,24 @@ variable "launch_type" {
   }
 }
 
+variable "ecs_service_timeouts" {
+  description = <<EOF
+  Default 20m. The timeouts for terraform update of the ECS service. If adjusted down, remember to also adjust health_check_grace_period_seconds.
+  Normally this should allow the ECS to at least do one retry of starting the container before timing out. The timeout is the whole rollout, not only the container startup.
+  I.e. health_check_grace_period_seconds * 2 + a bit extra > ecs_service_timeouts. 
```

Comment:
Default 20m. If you decide to reduce this timeout, be sure to adjust the `health_check_grace_period_seconds` accordingly. To ensure proper timing, the value of `health_check_grace_period_seconds` should be double, plus a little extra, compared to the `ecs_service_timeouts`.

Typically, the configuration should allow the ECS service to attempt at least one retry of container startup before the process times out. It's important to note that the timeout applies to the entire rollout process, not just the startup of the container.

Reply:
Mulig det blir litt langt